fdbclient/NativeAPI.actor.cpp

```diff
@@ -7619,7 +7643,7 @@ ACTOR Future<std::pair<Optional<StorageMetrics>, int>> waitStorageMetrics(Databa
 					TraceEvent(SevError, "WaitStorageMetricsError").error(e);
 					throw;
 				}
-				cx->invalidateCache(Key(), keys);
+				cx->invalidateCache(locations[0].tenantEntry.prefix, keys);
```

Comment:
Is it guaranteed here that `locations` in non-empty?

Reply:
I believe so, because within the `try` block (above), the `if` case is when `locations` has > 1 entry, and the `else` case references `locations[0]`.